src/Kiota.Builder/Refiners/PythonRefiner.cs

```diff
@@ -182,6 +182,18 @@ private static void CorrectCommonNames(CodeElement currentElement)
             {
                 param.Name = param.Name.ToFirstCharacterLowerCase().ToSnakeCase();
             }
+            if (parentClassM.IsOfKind(CodeClassKind.Model))
+            {
+                foreach (var prop in parentClassM.Properties)
```

Comment:
actually, we should probably filter on property of custom kind here, we probably don't want to rename additional data/backing store properties

Reply:
We could do that, but all model properties should be snake case, so technically there's no downside